homeworks/nikita.karchinskiy_Nikkarchinskiy/6-typescript/src/index.ts

```diff
@@ -0,0 +1,69 @@
+const letterBox = document.querySelector<HTMLDivElement>('[data-letter-task]')!;
+const scoreBox = document.querySelector<HTMLDivElement>('[data-score]')!;
+const scoreDif = document.querySelector<HTMLDivElement>('[data-circle]')!;
+const mainWrapper = document.querySelector<HTMLDivElement>('[data-main-wrapper]')!;
+
+let score: number = 100;
+let letter: string;
```

Comment:
There's no default value, so the variable equals `undefined` at the start. I consider using here default value, for example an empty string